internal/common/connectionsapi/client.go

```diff
@@ -27,9 +27,6 @@ const (
 
 func NewClient(authToken string, rawURL string, client *http.Client) (*Client, error) {
 	parsedURL, err := url.Parse(rawURL)
-	if parsedURL.Scheme != "https" {
```

Comment:
Removing this check in order to mock with normal httptest server in acceptance tests.